wordclock/sntp_client.c

```diff
@@ -46,7 +46,7 @@ void sntp_tsk(void *pvParameters)
 
 	/* Start SNTP */
 	printf("Starting SNTP... \n");
-	/* SNTP will request an update each 15 minutes */
+	/* SNTP will request an update each 5 minutes */
```

Comment:
Wijzigingen in deze file waren alleen voor debuggen. Als het is opgelost, mogen deze wijzigingen er weer uit. Deze file hoeft niet mee.

Reply:
Er stond nog veel meer in, maar dit leek me wel handig om er in te houden. als je dat van niet vind haal ik het er weer uit